README.md

```diff
@@ -1,32 +1,66 @@
 # Outil de facilitation du suivi budgétaire et comptable
 
+Ce dépôt contient deux parties : 
+- Une application qui récupère et formatte les données sur le Google Spreadsheet (le "back")
+- Une application qui présente les données pour qu'elle soient utiles aux utilisateurs de l'outil (le "front")
 
-Ce dépôt contient deux parties
+## Avant propos: les variables d'environnement
+Créez un fichier `.env` à la racine avec les informations suivantes:
+```txt
+API_URL=http://localhost:8000/api/ # ou l'URL sur laquelle pointe l'api de votre backend
+GOOGLE_API_KEY=[clé d'API developers.google.com]
```

Comment:
L'environnement étant partagé avec le front, j'ai l'impression que ce secret est partagé avec le monde entier, non ?

Reply:
Le fichier `.env` avec les valeurs de développement pourrait être ajouté.
La ligne avec `GOOGLE_API_KEY` pourrait être en commentaire (cette variable peut continuer à être indiqué via la ligne de commande).

La référence à `https://console.developers.google.com` me semble pertinente à conserver pour permettre aux contributeurs de récupérer une clé d'API.

Reply:
La doc de `dotenv-webpack` indique : 

> Your .env files can include sensitive information. Because of this,dotenv-webpack will only expose environment variables that are explicitly referenced in your code to your final bundle.
https://www.npmjs.com/package/dotenv-webpack

Mes tests confirme cela, cependant, il se peut que j'aie un "blind spot". Je veux bien qu'on regarde ensemble si cela est le cas.

Pour ce qui est du deuxième commentaire, est-ce que cela te satisferait : 
```suggestion
GOOGLE_API_KEY=[clé d'API à générer sur console.developers.google.com]
```